website/content/docs/api-gateway/install.mdx

```diff
@@ -0,0 +1,86 @@
+---
+layout: docs
+page_title: Consul API Gateway Install
+description:  >-
+  Installing Consul API Gateway
+---
+
+# Installing Consul API Gateway
+
+This topic describes how to use the Consul API Gateway add-on module. It includes instructions for installation and configuration.
+
+## Requirements
+
+Ensure that the environment you are deploying Consul API Gateway in meets the requirements listed in the [Technical Specifications][tech-specs]. This includes validating that the requirements for minimum versions of software are met. See the [Release Notes][rel-notes] for the version of API Gateway you are deploying.
+
+## Installation
+
+1. Set the version of Consul API Gateway you are installing as an environment variable. The following steps use this environment variable in commands and configurations.
+
+```shell-session
+$ export VERSION=0.3.0
+```
+
+1. Issue the following command to install the CRDs:
+
+```shell-session
+$ kubectl apply --kustomize="github.com/hashicorp/consul-api-gateway/config/crd?ref=v$VERSION"
+```
+
+1. Create a `values.yaml` file for your Consul API Gateway deployment by copying the following content and running it in the environment where you set the `VERSION` environment variable. The Consul Helm chart uses this `values.yaml` file to deploy the API Gateway. Available versions of the [Consul](https://hub.docker.com/r/hashicorp/consul/tags) and [Consul API Gateway](https://hub.docker.com/r/hashicorp/consul-api-gateway/tags) Docker images can be found on DockerHub, with additional context on version compatibility published in [GitHub releases](https://github.com/hashicorp/consul-api-gateway/releases). For more options to configure your Consul API Gateway deployment through the Helm chart, refer to [Helm Chart Configuration - apiGateway](https://www.consul.io/docs/k8s/helm#apigateway).
+
+<CodeBlockConfig filename="values.yaml">
+
+  ```shell
+  cat <<EOF > values.yaml
+    global:
+      name: consul
+    connectInject:
+      enabled: true
+    controller:
+      enabled: true
+    apiGateway:
+      enabled: true
+      image: hashicorp/consul-api-gateway:$VERSION
+  EOF
+  ```
+
+</CodeBlockConfig>
```

Comment:
since people are running the command rather than creating an actual file

````suggestion
<CodeBlockConfig>

```shell-session
$ cat <<EOF > values.yaml
    global:
      name: consul
    connectInject:
      enabled: true
    controller:
      enabled: true
    apiGateway:
      enabled: true
      image: hashicorp/consul-api-gateway:$VERSION
EOF
```

</CodeBlockConfig>
````